### PR TITLE
Restaurtant website is ready for production.

### PR DESCRIPTION
**Pull request by emasdev**
- add home layout
- configure css and js into webconfig
- add navbar functionallity
- generate html from javascript
- generate header
- generate home_page
- generate menu_page
- generate contact_page
- add transitions
- Fix linter errors
- Add github actions

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @emasdev  👋

**Good job so far here, nice folder structure also. 👌**

However let's make a couple of changes.

- [x] Please include an index.html under the `dist` directory and in the script section of the index.html link the main.js. This would provide a way to easily view the project, as one would simply need to run webpack to bundle the modules into main.js, and then by clicking index.html one can then view the project. Kindly include this.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/app/app.js [hunk 1]
```diff
@@ -1,101 +1,98 @@
-import { playTransition } from "./util/animate-transitions";
+import playTransition from './util/animate-transitions';
 
-export const run = () => {
+export default function run() {
```
Comment (mikenath223) [inline src/app/app.js]: - [x] Please change this to es6 arrow function syntax, Fix this and **other similar regular functions too**.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @emasdev :hand: 
Awesome job on this project so far. But we still need to make some changes before moving on.

- [x] The index.html file is missing from your project. We need to add the index.html file to the dist folder. Remember that the index.html should link the main.js file.

- [x] Remember that in the index.html we should append the HTML elements to the `<div>` with `id=content`. So we need to have at least one div element in the index.html file with the `id=container`.

- [ ] The live link at the README file is a broken link. We need to either fix it (recommended approach) or remove it.

Happy Coding :computer: 
**Sérgio Torres**

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :clap: :+1: 
Hi again @emasdev :hand: 
You may merge the PR.
Good luck with the next Project.

Happy Coding :computer: 
**Sérgio Torres**

**Pull request merged**